sites/funmooc/src/backend/static/richie/css/main.css

```diff
@@ -0,0 +1,18 @@
+/* Error: Top-level selectors may not contain the parent selector "&".
```

Comment:
I didn't get why this file and why only for funmooc?

Reply:
The SASS output directory build was wrong.